src/EditorFeatures/CSharpTest/EmbeddedLanguages/Json/CSharpJsonTests.cs

```diff
@@ -117,7 +117,7 @@ private void TryParseSubTrees(string stringText, bool strict)
                 return (token, null, allChars);
             }
 
-            var tree = JsonParser.TryParse(allChars, strict);
+            var tree = JsonParser.TryParse(allChars, strict ? JsonOptions.Strict : JsonOptions.None);
```

Comment:
the option to JsonParser became an enum so it can work with the new "detect language from comment" helper.